File: CPP/Program/sampahOrganik.cpp
#include <iostream>
#include <string>

using namespace std;

class SampahOrganik : public Sampah {
    // Atribut private
    private:
        string jenisOrganik; // Jenis sampah (sisa makanan, buah, Dll).
    
    public:
        // constructor kosong
        SampahOrganik() {
        }

        // Constructor
        SampahOrganik(string idSampah, string nama, string berat, 
            string jenisOrganik) 
            : Sampah(idSampah, nama, berat) {
            this->jenisOrganik = jenisOrganik;
        }
        
        // getter setter jenis Organik
        string getJenisOrganik() {return jenisOrganik;}

        void setJenisOrganik(string jenisOrganik) {this->jenisOrganik = jenisOrganik;}
        
    // Destructor
    ~SampahOrganik() {
    }
};

File: PYTHON/sampah_organik.py
from sampah import Sampah

class SampahOrganik(Sampah):
    def __init__(self, id_sampah="", nama="", berat="", jenis_organik=""):
        super().__init__(id_sampah, nama, berat)
        self.__jenis_organik = jenis_organik

    # getter setter jenis organik
    def get_jenis_organik(self):
        return self.__jenis_organik

    def set_jenis_organik(self, jenis_organik):
        self.__jenis_organik = jenis_organik
